src/core_codemods/combine_isinstance_issubclass.py

```diff
@@ -0,0 +1,99 @@
+import libcst as cst
+from libcst import matchers as m
+
+from codemodder.codemods.utils import extract_boolean_operands
+from codemodder.codemods.utils_mixin import NameResolutionMixin
+from core_codemods.api import Metadata, ReviewGuidance, SimpleCodemod
+
+
+class CombineIsinstanceIssubclass(SimpleCodemod, NameResolutionMixin):
+    metadata = Metadata(
+        name="combine-isinstance-issubclass",
+        summary="Simplify Boolean Expressions Using `isinstance` and `issubclass`",
+        review_guidance=ReviewGuidance.MERGE_WITHOUT_REVIEW,
+        references=[],
+    )
+    change_description = "Use tuple of matches instead of boolean expression with `isinstance` or `issubclass`"
+
+    def leave_BooleanOperation(
+        self, original_node: cst.BooleanOperation, updated_node: cst.BooleanOperation
+    ) -> cst.CSTNode:
+        if self.matches_isinstance_issubclass_or_pattern(original_node):
+            if not self.filter_by_path_includes_or_excludes(
```

Comment:
move this filter check up higher or combine if statements somehow.

Reply:
Sounds good I'll look at the core_codemods/docs and make these changes later today. 

Thanks for checking this out!